content/en/docs/tasks/security/authorization/authz-td-migration/index.md

```diff
@@ -112,7 +112,7 @@ Notice that it may take tens of seconds for the authorization policy to be propa
     Istio mesh is now running with a new trust domain, `new-td`.
 
 1. Delete secrets of `sleep` and `httpbin` in `default` namespace and in `sleep-allow` namespace. Notice if you install Istio with SDS,
```

Comment:
It's probably good to explain what does (install Istio with SDS) means. You may need to differentiate the case for step 1 (i.e should users use different profile or flags to change the TD value).

Reply:
This part is removed as no mounted secrets in the new release.